figures/scripts/figS2-snv-callers-panels.R

```diff
@@ -185,17 +204,19 @@ for (dataset in c("tcga", "pbta")) {
 
   ## VAF correlation plots --------------------------------------------
   print("Making VAF correlation plot")
-  # Correlate VAFs across callers pbta
-  cor_vaf <- all_caller_df %>%
-    select(starts_with("VAF_")) %>%
-    # 1:n() column for spread/gather bookkeeping which we do to remove `VAF_` from strips in this plot
-    mutate(n = 1:n()) %>%
-    # we want to remove
-    gather(caller, vaf, -n) %>%
-    mutate(caller = str_replace_all(caller, "VAF_", "")) %>%
-    spread(caller, vaf) %>%
-    select(-n) %>%
-    GGally::ggpairs(aes(alpha = 0.05)) +
+  # Correlate VAFs across callers
+  vaf_mat_df <- as.data.frame(detect_mat) %>%
+    # reorder columns; everything() will cover vardict which is only in PBTA
+    select(VAF_lancet, VAF_mutect, VAF_strelka, everything())
```

Comment:
Just want to note that this was the unneeded pivoting that I purged, and it had nothing to do with `index`.

Reply:
^ this says is outdated now, but it's not in terms of the spirit of my "fyi comment"